src/Layout/Presentation/DistrictOverview/DistrictOverview.tsx

```diff
@@ -44,15 +47,7 @@ export class DistrictOverview extends React.Component<DistrictOverviewProps, {}>
                         {leastWeightedDistrict}
                         {" hadde minst vekt, og bare telte "}
                         {lowestVsAverageInPercentage.toFixed(decimals) + "%."}
-                        {" Det mest utsatte sistemandatet (relativt til kvotient) var i "}
-                        {<span className="has-text-warning">{mostVulnerable.district}</span>}
-                        {" og ble vunnet av "}
-                        {mostVulnerable.winner.partyCode}
-                        {". "}
-                        {mostVulnerable.runnerUp.partyCode}
-                        {" ville trengt "}
-                        {mostVulnerable.moreVotesToWin.toFixed(0)}
-                        {" flere stemmer for å vinne det."}
+                        {this.getVulnerableText(mostVulnerable)}
```

Comment:
Likely a nice, clean change. Though we might consider simply turning it into a component instead.

Reply:
Fixed.